### PR TITLE
Bump rdkit version (#1304)

### PR DESCRIPTION
**Pull request by daud99**
* replace deprecated rdkit-pypi with rdkit package

* pin rdkit version to 2023.9.1

Thank you for taking your time to contribute to Ersilia, just a few checks before we proceed
- [ x] Have you followed the guidelines in our [Contribution Guide](https://github.com/ersilia-os/ersilia/blob/master/CONTRIBUTING.md)
- [x ] Have you written new tests for your core changes, as applicable?
- [ x] Have you successfully ran tests with your changes locally?

Related to #1304

### CODE REVIEW

**Comment by daud99** [comment]
@DhanshreeA 

* I have made the respective change.
* I also run the test locally, it all seems good.

Let me know your feedback.

File: ersilia/setup/requirements/compound.py [hunk 1]
```diff
@@ -11,7 +11,8 @@ def __init__(self):
             self.install()
 
     def install(self):
-        run_command("python -m pip install rdkit-pypi")
+        version = "2022.9.5"
```
Comment (DhanshreeA) [inline ersilia/setup/requirements/compound.py:14]: Where is this variable being used and it is also different from the version we are installing.

**Review by DhanshreeA — CHANGES_REQUESTED** [review]

File: ersilia/setup/requirements/compound.py (hunk 1 above)
Comment (daud99) [reply ersilia/setup/requirements/compound.py]: @DhanshreeA Sorry I didn't noticed this variable. Now, it is fixed!

**Comment by Malikbadmus** [comment]
@daud99 , Great job!!

@DhanshreeA , I've tested the code, and everything works as expected. It looks good to be merged.

**Review by DhanshreeA — APPROVED** [review]

**Pull request merged**